views/pages/contact.ejs

```diff
@@ -0,0 +1,32 @@
+<% include ../partials/head %>
+<div style="margin: 20px auto; width:50%; border-color: #CCCCCC;" class="border p-3 rounded clearfix" >
+<form>
+    <div class="form-group row">
+        <label for="colFormLabelSm" class="col-sm-2 col-form-label col-form-label-lg">To</label>
+        <div class="col-sm-10">
+            <select class="form-control" id="exampleFormControlSelect1">

```

Comment:
Users should be able to send messages to other users via username. They should also be able to send to admin, do this by having a button that fills out the word "Admin" under "To: "